src/components/activity-groups-list/activity-group-item/activity-group-amount-change/activity-group-amount-change.tsx

```diff
@@ -82,10 +79,11 @@ export const ActivityGroupAmountChange: FC<Props> = ({ group }) => {
           key={index}
           style={[
             styles.valueText,
-            conditionalStyle(amount.isPositive(), styles.positiveAmountText, styles.negativeAmountText)
+            conditionalStyle(amount > 0, styles.positiveAmountText, styles.negativeAmountText)
           ]}>
-          {amount.isPositive() ? '+ ' : '- '}
-          {amount.abs().toFixed()}
+          {Math.abs(amount) < 0.01 && '≈ '}
```

Comment:
create `minAmountValue` & `maxAmountValue` variables and use 
`import { inRange } from 'lodash-es';`